src/deprecation.h

```diff
@@ -8,10 +8,9 @@
 // Deprecation policy:
 // * Shut down 16 weeks' worth of blocks after the estimated release block height.
 // * A warning is shown during the 2 weeks' worth of blocks prior to shut down.
-static const int APPROX_RELEASE_HEIGHT = 840277;
-// static const int WEEKS_UNTIL_DEPRECATION = 16;
-// static const int DEPRECATION_HEIGHT = APPROX_RELEASE_HEIGHT + (WEEKS_UNTIL_DEPRECATION * 7 * 24 * 48);
-static const int DEPRECATION_HEIGHT = 973713;
+static const int APPROX_RELEASE_HEIGHT = 901000;
+static const int WEEKS_UNTIL_DEPRECATION = 16;
+static const int DEPRECATION_HEIGHT = APPROX_RELEASE_HEIGHT + (WEEKS_UNTIL_DEPRECATION * 7 * 24 * 48);
```

Comment:
This gives me an EoS halt height of 1030024, which is before the targeted Canopy mainnet activation height of 1046400.

Reply:
Thanks for remembering to check this! :-)